example/src/App.tsx

```diff
@@ -181,14 +183,33 @@ class App extends React.Component<{}, State> {
         key: 'status',
         sort: true,
         style: { width: '150px' },
-        injectBody: (value: string) => <Badge status={value === 'Published' ? 'success' : 'warning'}>{value}</Badge>,
+        injectBody: (value: any) => <Badge status={value.status === 'Published' ? 'success' : 'warning'}>{value.status}</Badge>,
       }, {
         label: 'Type',
         key: 'type',
         style: { width: '100px' },
       },
     ];
 
+    /* 
+      Filtering table data
+      mode: Mode of filteration, right now we have only one which is search
+      search: If mode is search then it holds search config
+        event: When the search should fire, on click or on input
+        field: Name of field by which search should happen
+        placeholder: Filed placeholder
+        title: Title of field
+    */
+    const filterConfig = {
```

Comment:
@dheeraja00 @weegrom The use case is that the filter textbox can be elsewhere on the page, in an entirely different segment.  In which case, it would be better if we just pass a filterQuery property to the table.